### PR TITLE
[sdks] Instead of a XCODE_VERSION variable, use a XCODE_DEVELOPER_ROOT variable which contains the full path to xcode, since this is what x.ios is using.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by marek-safar** [comment]
Please don't forget to backport it

**Comment by vargaz** [comment]
@monojenkins backport master

File: sdks/paths.mk [hunk 1]
```diff
@@ -43,7 +43,7 @@ ifndef DISABLE_IOS
 CheckXcodeDir=$(or $(and $(wildcard $(1))),$(warning Could not find Xcode in "$(1)"))
 
 ifneq ($(call CheckXcodeDir,/Applications/Xcode.app/Contents/Developer),)
-XCODE_DIR=/Applications/$(XCODE_VERSION).app/Contents/Developer
+XCODE_DIR=$(XCODE_DEVELOPER_ROOT)
```
Comment (akoeplinger) [inline sdks/paths.mk]: the `CheckXcodeDir` if above should be changed to check `$(XCODE_DEVELOPER_ROOT)`  too

File: sdks/versions.mk [hunk 2]
```diff
@@ -17,8 +17,7 @@ ANDROID_PLATFORM_VERSION_x86_64?=21
 
 # iOS
 
-## Xcode needs to be installed in /Applications/$(XCODE_VERSION).app
-XCODE_VERSION?=Xcode
+XCODE_DEVELOPER_ROOT?=/Applications/Xcode.app/Contents/Developer
```
Comment (luhenry) [inline sdks/versions.mk]: Can't we just define `XCODE_DIR` here then?

**Pull request merged**